### PR TITLE
Improve error message for retrieving data key

### PR DESCRIPTION
**Pull request by autrilla**
What it looks like (AWS data scrubbed)

This is how it looks

### CODE REVIEW

File: usererrors.go [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+		strings.Join(groupErrs, "\n\n"), trailer)
+}
+
+type decryptGroupError struct {
+	groupName string
+	err       error
+}
+
+func (r *decryptGroupError) Error() string {
+	return fmt.Sprintf("could not decryt group %s: %s", r.groupName, r.err)
```
Comment (jvehent) [inline usererrors.go]: type: decryt

File: usererrors.go [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+	header := fmt.Sprintf(`Group %s: %s`, r.groupName, status)
+	if r.err == nil {
+		return header
+	}
+	message := r.err.Error()
+	if userError, ok := r.err.(UserError); ok {
+		message = userError.UserError()
+	}
+	reader := prefixer.New(strings.NewReader(message), "  ")
+	errMsg, _ := ioutil.ReadAll(reader)
```
Comment (jvehent) [inline usererrors.go]: maybe shouldn't ignore the error here?

File: usererrors.go [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+	}
+	header := fmt.Sprintf("%s: %s", e.keyName, status)
+	if e.isSuccessful() {
+		return header
+	}
+	var errMessages []string
+	for _, err := range e.errs {
+		wrappedErr := wordwrap.WrapString(err.Error(), 60)
+		reader := prefixer.New(strings.NewReader(wrappedErr), "  | ")
+		errMsg, _ := ioutil.ReadAll(reader)
```
Comment (jvehent) [inline usererrors.go]: ignored error

**Comment by autrilla** [comment]
I suppose the coverage decrease is expected -- can't really test error messages meaningfully.

**Pull request merged**